src/main/java/seedu/module/ui/ModuleCard.java

```diff
@@ -18,8 +18,6 @@
      * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
      * As a consequence, UI elements' variable names cannot be set to such keywords
      * or an exception will be thrown by JavaFX during runtime.
-     *
-     * @see <a href="https://github.com/se-edu/modulebook-level4/issues/336">The issue on ModuleBook level 4</a>
```

Comment:
I think this link can be kept, but it has to be changed back to AddressBook level 4 since that was the original project from which this came out.